### PR TITLE
add a xmas tree to candor

### PR DESCRIPTION
**Pull request by xdeimos**

### CODE REVIEW

File: world/map/npc/annuals/tree_beard.txt [hunk 1]
```diff
@@ -45,7 +46,24 @@
...
+}
+
+029-1,52,94,0|script|Present#4|386
+{
+    callfunc "XmasStates";
+    callfunc "XmasMainItemReward";
+    close;
+}
+
+001-1,58,75,0|script|#GhostTreeThree|133
```
Comment (mekolat) [inline world/map/npc/annuals/tree_beard.txt]: why did you move `#GhostTreeThree` ?

**Comment by mekolat** [comment]
@wushin since we add a tree to candor maybe we could put an event portal there too?

**Comment by wushin** [comment]
Do we really want to suck nubs up into GM events?
